src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
+
+import { useDispatch } from 'react-redux';
+
+import Rocketcontainer from '../rocket/rocketcontainer';
+import MissionCard from '../mission/missionCard';
+import Profile from '../myProfile/Profile';
+import DragonPage from '../dragon/dragon';
+import { loadDragons } from '../../redux/dragon/dragon';
+import { fetchRockets } from '../../redux/rocket/rocketSlice';
+import { fetchMissions } from '../../redux/mission/missionSlice';
+
+function Navbar() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
+
+  const getDragons = () => {
+    dispatch(loadDragons());
+  };
+  useEffect(() => {
+    getDragons();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, [dispatch]);
+  return (
+    <div>
+      <header className="panel-bg">
+        <div className="navigation">
+          <div className="logoFrame">
+            <img className="logo" src="/planet.png" alt="space logo" />
+            <span className="Logo-Text">Space Travelers Hub</span>
+          </div>
+          <ul>
+            <li>
+              <Link to="/" className="Mission Text-Style-3">
```

Comment:
- [ ] As required by project requirements, I kindly suggest that you use `Navlink`. The Link component is used to navigate the different routes on the site. But `NavLink` is used to add the style attributes to the active routes.
`NavLink` will make it easier for you to style your Links and it is a requirement for active links to have an underline.
- [ ] As required. I kindly suggest that your current active page is indicated with an underline in the `NavLink`.



| Your Design | Figma Design |
| ------------- | ------------- |
| <img width="342" alt="image" src="https://user-images.githubusercontent.com/52934128/196004227-8de4126c-20d5-4639-9acf-b16fe124b21e.png">  | <img width="388" alt="image" src="https://user-images.githubusercontent.com/52934128/196004277-be9dd8dc-4e01-4f47-a9a5-cf16627748d6.png">  |

```
 <ul>
        <li>
          <NavLink exact activeClassName="active" to="/">
            Home
          </NavLink>
        </li>
        <li>
          <NavLink activeClassName="active" to="/users">
            Users
          </NavLink>
        </li>
        <li>
          <NavLink activeClassName="active" to="/contact">
            Contact
          </NavLink>
        </li>
      </ul>
```

Reply:
Thank you for the review. Resolved.